pkg/controller/installation/products/rhsso/reconciler.go

```diff
@@ -286,6 +295,41 @@ func (r *Reconciler) setupOpenshiftIDP(ctx context.Context, kcr *aerogearv1.Keyc
 	return nil
 }
 
+func (r *Reconciler) setupGithubIDP(ctx context.Context, kcr *aerogearv1.KeycloakRealm, serverClient pkgclient.Client) error {
+	githubCreds := &v1.Secret{}
+	err := serverClient.Get(ctx, pkgclient.ObjectKey{Name: githubOauthAppCredentialsSecretName, Namespace: r.ConfigManager.GetOperatorNamespace()}, githubCreds)
+	if err != nil {
+		logrus.Errorf("Unable to find Github oauth credentials secret in namespace %s", r.ConfigManager.GetOperatorNamespace())
+		return err
+	}
+
+	if !containsIdentityProvider(kcr.Spec.IdentityProviders, githubIdpAlias) {
+		logrus.Infof("Adding github identity provider to the keycloak realm %s %s ", githubCreds.Data["clientId"], githubCreds.Data["secret"])
```

Comment:
It's probably best not to log the secret, in case the log gets passed around and accidentally gets posted in public gist or smth.

Reply:
ah that was me testing, will remove those :+1: